allennlp/training/metrics/span_based_f1_measure.py

```diff
@@ -155,6 +155,14 @@ def _extract_spans(self, tag_sequence: List[int]) -> Set[Tuple[Tuple[int, int],
                 # We don't care about tags we are
                 # told to ignore, so we do nothing.
                 continue
+            elif bio_tag == "U":
```

Comment:
It'd be nice if the test for this made sure `U` tags were handled correctly - I don't think you updated the test to check for this.